src/backend/api/Fusion.Resources.Api/Authorization/Handlers/DelegatedContractRoleAuthHandler.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Fusion.Integration.Org;
+using Fusion.Resources.Database;
+using Fusion.Resources.Domain;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api.Authorization.Handlers
+{
+    internal class DelegatedContractRoleAuthHandler : AuthorizationHandler<DelegatedContractRole, ContractResource>
+    {
+        private readonly ResourcesDbContext dbContext;
+
+        public DelegatedContractRoleAuthHandler(ResourcesDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, DelegatedContractRole requirement, ContractResource resource)
+        {
+            var userId = context.User.GetAzureUniqueId();
+
+            // Cannot evaluate users without azure unique id.
+            if (userId.HasValue == false)
```

Comment:
It can happen, if the object id claim is not there. Granted, should not happen, but you never know.. But might as well just check, since it is a nullable type that is returned.